### DISCUSSION TITLE
Does O3DE Support Dynamic Materials for Round Progress Bar?

### DISCUSSION THREAD

**Discussion by Kraythax** [o3de/o3de#13837] [Q&A]
I worked a bit with Unreal and now I am looking at O3DE and I am wondering if it supports Dynamic Materials. I followed the tutorial on the following YouTube video, modified it for my needs and implemented the material. I would like to know if this is possible in O3DE and if so can anyone point me at videos, examples or documentation? The video is here: 

https://www.youtube.com/watch?v=_xI-YsxKS-4&list=PLaJFexnqYTv61tgb1SvRdHL_Y8gzExoPv&index=8

**Comment by AMZN-daimini** [comment]
@o3de/sig-graphics-audio

**Comment by santorac** [comment]
Material Canvas is a new tool currently in development for node graph material authoring. An early functional preview of the tool is available in the development branch. See https://github.com/o3de/sig-graphics-audio/issues/51
